how-to-team-desktop/issues.md

```diff
@@ -0,0 +1,85 @@
+
+**See [Labels](#labels) for labeling standards. Labeling doesn't only happen once, but is a continual process to document an issue's or PR's state.*
+
+**Definitions**
+
+| Word | Definition |
+|------|------------|
+| Duplicate | An open or closed issue already exists for the issue. |
+| Reproducible | The issue can be encountered by someone other than the opener. |
+| Stale | No activity has been received on an issue/PR within a set amount of time. Recommended time to be considered stale is 60 days, but can be variable depending on the project and issue. Generally should be automated. |
+
+## Labels
+
+*Edit repository labels at `https://github.com/<organization>/<repository>/labels`.*
+
+Individual repositories should use these labels in addition to repository-specific labels to further classify issues and pull requests. For example, the [Steam Snap](https://github.com/canonical/steam-snap/labels) has labels to classify an issue further such as `game-specific`, `client-specific`, `hardware-specific`, etc that are labels exclusive to the Snap. Many repositories may not need additional labels other than the ones defined below.
+
+### Types
+
+Generally an issue or PR should have exactly one of the following labels indiciating the type of the issue or PR. These may be automatically applied by using [issues templates](#templates) and *usually are never changed or removed once added*.
+
+| Label | Use Case |
+|-------|----------|
+| `type/bug` | Issues that are related to a problem or unexpected behavior. |
+| `type/documentation` | Issues or PRs relating to documentation or wiki pages. |
+| `type/enhancement` | Issues or PRs relating to new feature requests. |
+| `type/question` | Issues requesting information or to have a discussion. |
+
+### Triage States
+
+Labels to be applied while triaging an issue, indicating the result of triage. Usually these states lead to the issue or PR being closed once applied.
```

Comment:
If we go down the flow of "Enough Information?" -> "No" or "Reproducible" -> "No", it could be nice to have an additional `triage/needs-information` tag to pin the triage state.

Reply:
I thought about this too, and my only hangup is I wouldn't want that label to constantly be added/removed as you ask the openers questions.

It may be the case that you need info, the label gets added, you get info from the opener, the label gets removed. Then you may need more info down the line as you work to resolve the issue, so we need to add the label again. This all shows in the issue history as labels are added/removed, and it could get quite messy, so is there a good heuristic for when the label should be used and when not to use it? Should we only use that label for the initial triage and never used again?

Reply:
Fair point about having triage leading to a closing state. Does also keep the flow simple. Yea, having it as an additional tag sounds good :+1: